mail_activity_done/models/res_users.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018 Eficent <http://www.eficent.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+from odoo import api, fields, models, modules
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    @api.model
+    def systray_get_activities(self):
+        # Here we totally override the method. Not very nice, but
```

Comment:
Maybe we could add a context here to use the old method?

if self.env.context.get('use_old_systray_get_activities', False):
    return super().systray_get_activities